src/pages/settings/Profile/CustomStatus/StatusClearAfterPage.js

```diff
@@ -157,9 +157,9 @@ function StatusClearAfterPage({currentUserPersonalDetails, customStatus}) {
     const timePeriodOptions = useCallback(
         () =>
             _.map(statusType, (item, index) => (
-                <BaseListItem
+                <RadioListItem
                     item={item}
-                    key={`${index}+${item.value}`}
+                    keyForList={`${index}+${item.value}`}
```

Comment:
Looks like we don't need this
https://github.com/Expensify/App/blob/8889914886d7e7dead0fdfcac0495658649539a5/src/components/SelectionList/RadioListItem.tsx#L36

Reply:
Ah nice!